linux/gosv_testcase_list.yml

```diff
@@ -2,13 +2,13 @@
 # SPDX-License-Identifier: BSD-2-Clause
 ---
 - import_playbook: deploy_vm/deploy_vm.yml
+- import_playbook: check_inbox_driver/check_inbox_driver.yml
```

Comment:
Do we need to check/get driver info after ovt installed or updated?

Reply:
New change:
If no open-vm-tools is installed while running this case, and ovt_verify_install installs an ovt from OS image or OS official repo, that ovt version will be updated into os release info json file by log plubin.